shared/src/common/types/interview/interview-get-response-dto.type.ts

```diff
@@ -0,0 +1,12 @@
+import { CategoryGetAllItemResponseDto, UsersGetResponseDto } from '../types';
+
+type InterviewResponseDto = {
```

Comment:
file name should be interviews-by-id-response-dto.type.ts
```suggestion
type InterviewsByIdResponseDto = {
```